locales/en/moves.json

```diff
@@ -219,5 +219,6 @@
         "message": "$t(moves::update_flash.supplier_message)"
       }
     }
-  }
+  },
+  "redirect_notes": "Requested by {{fullname}} (user_name:{{username}} / user_id:{{userId}})"
```

Comment:
Is this something we play back to a user ever or just for our records?

I'm wondering if we _really_ need to send the person's username/user ID along with their name.

There will be a separate piece of work coming soon which will look at moving the auth to the API so that the API will know who does what.

And I wonder if we leave trying to send audit traces as part of "notes" until then and for now we just send the name which is the information we already have.

Reply:
Yes, the auth logging would indeed be better, but I've checked with Jason and he's happy with this.